modules/internal/src/test/scala/util/EncoderUtilTest.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2017-2018 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package freestyle.rpc
+package internal
+package util
+
+import org.scalatest._
+import org.scalacheck.Prop._
+import org.scalatest.prop.Checkers
+
+class EncoderUtilTest extends WordSpec with Matchers with Checkers {
+
+  "EncoderUtil" should {
+
+    "allow to convert int to and from byteArray" in {
+      check {
+        { n: Int =>
```

Comment:
here is missing the `forAll` right? maybe I'm missing something but I think this should be:
```scala
check {
  forAll { n: Int =>
    val value: Array[Byte] = EncoderUtil.intToByteArray(n)
    EncoderUtil.byteArrayToInt(value) == n
  }
}
```

Reply:
Yes Fede, you are right.

Reply:
Ok, we can tackle it in the next PR. Thanks